content/docs/SDKs/javascript/changelog.mdx

```diff
@@ -7,6 +7,16 @@ description: Release notes for the Agentuity JavaScript SDK
 
 This page contains the release notes for the Agentuity JavaScript SDK.
 
+## v0.0.89
```

Comment:
use the same heading format as the others with the date in the heading and the `[version]` around it